### PR TITLE
News

### PR DESCRIPTION
**Pull request by sparkdevelopment**
News customiser features ready to go

### CODE REVIEW

File: wp-content/themes/govintranetpress/functions.php [hunk 1]
```diff
@@ -1576,7 +1579,7 @@ function relevanssi_user_filter($hits) {
 
 // Force logout after 1 hour
 function keep_me_logged_in_for_1_hour( $expirein ) {
-    return 3600; // 1 hour in seconds
+    return 3600*8; // 1 hour in seconds
```
Comment (marcincichon) [inline wp-content/themes/govintranetpress/functions.php]: Just curious - why * 8?

File: wp-content/themes/govintranetpress/functions.php [hunk 2]
```diff
@@ -1621,3 +1624,24 @@ function add_markdown_to_cpts() {
...
+// Extra customiser controls
+require_once('inc/customiser-controls.php');
+
+// Allow editors to access theme options
+$role_object = get_role( 'editor' );
+$role_object->add_cap( 'edit_theme_options' );
+
+// Make sure autocomplete dropdown is not hidden by customizer
+function dw_customizer_styles() { ?>
+    <style>
```
Comment (marcincichon) [inline wp-content/themes/govintranetpress/functions.php]: Any chance we can move this to css?

File: wp-content/themes/govintranetpress/functions.php (hunk 1 above)
Comment (sparkdevelopment) [inline wp-content/themes/govintranetpress/functions.php]: I was getting annoyed with it logging me out after an hour. It's something
we need to decide on for production. Might have separate values for
admins/non-admins.
ᐧ

On 7 January 2015 at 11:05, Marcin Cichon <notifications@github.com> wrote:

> In wp-content/themes/govintranetpress/functions.php
> <https://github.com/ministryofjustice/mojintranet/pull/18#discussion-diff-22581003>
> :
>
> > @@ -1576,7 +1579,7 @@ function relevanssi_user_filter($hits) {
> >
> >  // Force logout after 1 hour
> >  function keep_me_logged_in_for_1_hour( $expirein ) {
> > -    return 3600; // 1 hour in seconds
> > +    return 3600*8; // 1 hour in seconds
>
> Just curious - why * 8?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/mojintranet/pull/18/files#r22581003>
> .
>



-- 
*Ryan Jarrett*
Lead Wordpress Developer | MOJ Digital Services | 10.51 102 Petty France
| London | SW1H 9AJ | 07961 300888

Comment (marcincichon) [inline wp-content/themes/govintranetpress/functions.php]: Agreed. Maybe we could use it for prod only?

File: wp-content/themes/govintranetpress/functions.php (hunk 2 above)
Comment (sparkdevelopment) [inline wp-content/themes/govintranetpress/functions.php]: I didn't want it to affect the rest of the site - this is only applied to
the customizer page
ᐧ

On 7 January 2015 at 11:05, Marcin Cichon <notifications@github.com> wrote:

> In wp-content/themes/govintranetpress/functions.php
> <https://github.com/ministryofjustice/mojintranet/pull/18#discussion-diff-22581020>
> :
>
> > @@ -1621,3 +1624,24 @@ function add_markdown_to_cpts() {
> >  	}
> >  }
> >  add_action('init', 'add_markdown_to_cpts');
> > +
> > +// Setup news customiser
> > +require_once('inc/news-customiser.php');
> > +// Extra customiser controls
> > +require_once('inc/customiser-controls.php');
> > +
> > +// Allow editors to access theme options
> > +$role_object = get_role( 'editor' );
> > +$role_object->add_cap( 'edit_theme_options' );
> > +
> > +// Make sure autocomplete dropdown is not hidden by customizer
> > +function dw_customizer_styles() { ?>
> > +    <style>
>
> Any chance we can move this to css?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/mojintranet/pull/18/files#r22581020>
> .
>



-- 
*Ryan Jarrett*
Lead Wordpress Developer | MOJ Digital Services | 10.51 102 Petty France
| London | SW1H 9AJ | 07961 300888

File: wp-content/themes/govintranetpress/functions.php (hunk 1 above)
Comment (sparkdevelopment) [inline wp-content/themes/govintranetpress/functions.php]: Yeah - I think there's value in auto-logging out. Perhaps set a flag in
production wp-config and if it's set then set the cookie? That way we can
leave this code in the repo (once modified) but wp-config isn't versioned
so we can maintain a separate config that way
ᐧ

On 7 January 2015 at 11:08, Marcin Cichon <notifications@github.com> wrote:

> In wp-content/themes/govintranetpress/functions.php
> <https://github.com/ministryofjustice/mojintranet/pull/18#discussion-diff-22581116>
> :
>
> > @@ -1576,7 +1579,7 @@ function relevanssi_user_filter($hits) {
> >
> >  // Force logout after 1 hour
> >  function keep_me_logged_in_for_1_hour( $expirein ) {
> > -    return 3600; // 1 hour in seconds
> > +    return 3600*8; // 1 hour in seconds
>
> Agreed. Maybe we could use it for prod only?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/mojintranet/pull/18/files#r22581116>
> .
>



-- 
*Ryan Jarrett*
Lead Wordpress Developer | MOJ Digital Services | 10.51 102 Petty France
| London | SW1H 9AJ | 07961 300888

Comment (marcincichon) [inline wp-content/themes/govintranetpress/functions.php]: Sounds good

**Pull request merged**